src/components/HomeButtons.tsx

```diff
@@ -130,18 +184,16 @@ const HomeButtons = () => (
   <>
     {buttons.map((button: THomeButton) => {
       return (
-        <Grid item xs={6} key={button.text}>
-          <HomeLink to={button.to} data-test={button.text}>
-            <HomeButton
-              disableRipple={true}
-              component={'span'}
-              buttonColor={button.color}
-            >
-              {button.text}
-              {button.icon}
-            </HomeButton>
-          </HomeLink>
-        </Grid>
+        <HomeLink {...button} key={button.text}>
+          <HomeButton
+            disableRipple={true}
```

Comment:
why?

Reply:
Users won't be able to see the ripple since they're instantly routed to a different page. Since it doesn't really matter, I just [removed the prop](https://github.com/CodeForBoulder/upswyng/pull/87/commits/a3fdf2968081252088f3a06cb28f054a92fe9639).